tools/python/boutiques/validator.py

```diff
@@ -22,6 +23,9 @@ def validate_descriptor(json_file, **kwargs):
     """
     path, fil = op.split(bfile)
     schema_file = op.join(path, "schema", "descriptor.schema.json")
+    allowed_keywords = ['and', 'or', 'false', 'true']
```

Comment:
Because capitalization may be funny/distinct across representations (in particular `True` versus `true`), we may want to either a) add more capitalizations to this list, or b) convert everything to an upper/lowercase value when we compare. Essentially just an extra precaution to avoid the hole of evaluating a JSON `true` and a boolean Python `True` to be distinct.

Reply:
I opted for option b), just in case someone needs to write ```ANd``` :smile: